terraformignore.go

```diff
@@ -200,9 +206,13 @@ var defaultExclusions = []rule{
 		excluded: false,
 	},
 	{
-		val:      filepath.Join("**", ".terraform", "**"),
+		val:      filepath.Join("**", ".terraform", "?**"),
```

Comment:
In order to actually discovery `.terraform/modules`, we have to explicitly allow recursion into `.terraform` but not anything under it.